wharehauora-wifi-gateway/wharehauora-wifi-gateway.ino

```diff
@@ -6,11 +6,32 @@
 
 const char my_ssid[] = "";
 const char my_pass[] = "";
-const char my_server[] = "m11.cloudmqtt.com";
+
+#ifdef STAGING
```

Comment:
#ifdef block is new to me in C. Interesting!

Reply:
ah! I should probably put an example in there.

Reply:
I've added a `#define STAGING` up on line 7  - to make it clearer

Reply:
👍